src/services/messages-processor/service.ts

```diff
@@ -269,11 +284,35 @@ export const makeMessagesProcessor = ({
     }
   }
 
-  const readFolder = async (uri: string): Promise<string[]> => {
+  const readFolder = async (uri: string): Promise<MessageFile[]> => {
     if (uri.startsWith('s3://')) return s3Service.read(uri)
     if (uri.startsWith('gs://')) return gsService.read(uri)
     return localFileReader.readFilesFromFolder(uri)
   }
 
-  return { load, verify, exit }
+  const loadToMemoryStorage = async (
+    messagesStorage: MessageStorage
+  ): Promise<{
+    updated: number
+    added: number
+  }> => {
+    invalidExitMessageFiles.clear()
+
+    const newMessages = await load(messagesStorage)
```

Comment:
Perhaps `load` -> `loadNewMessages` would be clearer

Reply:
Renamed